benchclients/python/tests/test_conbench_client.py

```diff
@@ -103,12 +102,21 @@ def test_cc_performs_login_when_env_is_set(
     }
 
     httpserver.expect_oneshot_request(
-        "/api/login/", method="POST", json=creds
```

Comment:
I assumed the redirection 'just works', but that was a wrong assumption: https://github.com/conbench/conbench/pull/1150